src/rg3d/beginning/getting_started.md

```diff
@@ -7,7 +7,7 @@ To begin using the engine, just add following lines to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rg3d = { git = "https://github.com/rg3dengine/rg3d" }
+rg3d = { git = "https://github.com/rg3dengine/rg3d" } 
```

Comment:
It seems that it is not needed anymore

Reply:
I deleted it before PR. You still see it?

Reply:
Yeah, the file is still changed, I think you can revert changes from it.